src/api/v1/paths/doughs/{doughId}.js

```diff
@@ -0,0 +1,28 @@
+import { getDoughById } from 'api/v1/db/oracledb/doughs-dao';
+import { errorBuilder, errorHandler } from 'errors/errors';
+
+/**
+ * Get a dough with id `req.params.doughId`
+ *
+ * @type {RequestHandler}
+ */
+const get = async (req, res) => {
+  try {
+    const result = await getDoughById(req.params.doughId);
+    if (result.data.length === 0) {
```

Comment:
No need to change, but I personally prefer using `!result.data` or [`_.isEmpty`](https://lodash.com/docs/4.17.15#isEmpty) to check if an array is empty.

Reply:
`!result.data` wasn't working for whatever reason, but `_.isEmpty` looks like a good idea.

Reply:
Hmm, can you investigate why `!result.data` didn't work?

```js
> data = []
> !data
false
```

Reply:
My bad, looks like [`[]` is trythy in JavaScripts](https://guide.freecodecamp.org/javascript/falsy-values/). Weird language.

Reply:
Yeah, that behavior actually kinda makes sense in my opinion but it's definitely not consistent with the rest of the language.